src/foam/dao/DAOProperty.js

```diff
@@ -41,7 +41,19 @@ foam.CLASS({
       value: 'HIDDEN'
     },
     ['transient', true],
-    ['of', 'foam.dao.DAO'],
+    {
```

Comment:
Why would we even let you set the 'of' of a DAOProperty? Shouldn't it always just be 'DAO'? Couldn't we just override the setter to provide a warning that you can't set it?